e2e-polybft/e2e/bridge_test.go

```diff
@@ -535,17 +536,25 @@ func TestE2E_Bridge_ERC721Transfer(t *testing.T) {
 	// the transactions are processed and there should be a success events
 	var stateSyncedResult contractsapi.StateSyncResultEvent
 
-	logs, err := getFilteredLogs(stateSyncedResult.Sig(), 0, 50, childEthEndpoint)
-	require.NoError(t, err)
+	for i := 0; i < numberOfAttempts; i++ {
+		logs, err := getFilteredLogs(stateSyncedResult.Sig(), 0, uint64(50+i*epochSize), childEthEndpoint)
+		require.NoError(t, err)
+
+		if len(logs) == 2 || i == numberOfAttempts-1 {
```

Comment:
Since it is checked inside checkStateSyncResultLogs, I'd simplify this if statement:

```suggestion
		if i == numberOfAttempts-1 {
```